querybook/webapp/resource/dataDoc.ts

```diff
@@ -101,6 +101,8 @@ export const DataDocResource = {
         }>(`/favorite_data_doc/${docId}/`),
     unfavorite: (docId: number) => ds.delete(`/favorite_data_doc/${docId}/`),
 
+    run: (docId: number) => ds.save<null>(`/datadoc/${docId}/run/`),
```

Comment:
the return type is not null?

Reply:
we dont really care about the return value. I'll update the api to not return